server/__test__/test.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
When we use JavaScript files to test our routes, we won't be able use ES6 features like imports/exports, you need to modify it and use the previous configs.